src/main/java/com/metamx/emitter/core/HttpEmitterConfig.java

```diff
@@ -65,8 +68,16 @@
   @JsonProperty
   private ContentEncoding contentEncoding = DEFAULT_CONTENT_ENCODING;
 
+  /**
+   * @deprecated use {@link Builder}
+   */
+  @Deprecated
```

Comment:
Yes, the default constructor is for the easy deserialisation only. Just commented it.